chutney/server/src/main/java/com/chutneytesting/campaign/api/dto/CampaignMapper.java

```diff
@@ -30,7 +32,8 @@ public static CampaignDto toDtoWithoutReport(Campaign campaign) {
             campaign.id,
             campaign.title,
             campaign.description,
-            campaign.scenarioIds,
+            campaign.campaignScenarios.stream().map(Campaign.CampaignScenario::scenarioId).toList(),
```

Comment:
Check with all our client (i think there is only Jabir) if they use it.
If so keep two version
if not no need to keep retrocompatibility.

Reply:
Compatibility broken.